### DISCUSSION TITLE
OpenChain M&A Summit 2022 Video Record

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#93] [日常讨论]
The OpenChain M&A Summit 2022 provides an extraordinarily detailed dive into how open source - and standards like OpenChain ISO/IEC 5230 - play a pivotal part in modern Mergers and Acquisitions. Five talks, one panel. Freely available to all.

https://www.openchainproject.org/news/2022/05/01/openchain-ma-summit-2022

**Comment by wangleo61** [comment]
Looks very nice,

Thanks sharing Will


发件人: Will ***@***.***>
发送时间: Wednesday, May 18, 2022 6:25 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] OpenChain M&A Summit 2022 Video Record (Discussion #93)


The OpenChain M&A Summit 2022 provides an extraordinarily detailed dive into how open source - and standards like OpenChain ISO/IEC 5230 - play a pivotal part in modern Mergers and Acquisitions. Five talks, one panel. Freely available to all.

https://www.openchainproject.org/news/2022/05/01/openchain-ma-summit-2022

―
